Gemfile

```diff
@@ -3,7 +3,9 @@ source 'https://rubygems.org'
 # Specify your gem's dependencies in amazon_ssa_support.gemspec
 gemspec
 
+gem "log_decorator", ">0", :require => 'log_decorator', :git => "https://github.com/ManageIQ/log_decorator.git", :branch => "master"
```

Comment:
`log_decorator` is a published gem, so you should remove the `:git =>` portion.

Reply:
Changed them accordingly.